corehq/motech/requests.py

```diff
@@ -215,3 +216,31 @@ def parse_request_exception(err):
     else:
         err_response = str(err)
     return err_request, err_response
+
+
+def simple_post(domain, url, data, *, headers, auth, verify,
+                timeout=60, notify_addresses=None, payload_id=None):
```

Comment:
Could probably drop the `timeout` param too, since it's set to a constant 75, and I'm guesing the 60 is arbitrary